eglib/configure.ac

```diff
@@ -151,7 +162,7 @@ elif test x$target_ios = xno; then
 AC_CHECK_FUNCS(strndup getpwuid_r)
 fi
 
-AM_CONDITIONAL(NEED_VASPRINTF, test x$ac_cv_func_vasprintf = xno )
+AM_CONDITIONAL(NEED_VASPRINTF, test x$ac_cv_func_vasprintf = xno || test x$with_overridable_allocators == xyes)
```

Comment:
The problem is that the system vasprintf allocates memory with malloc, which which is a problem if you then try to free that memory with a overridden g_free